README.md

```diff
@@ -107,3 +107,7 @@ Removes all associated links for a given EORI based on session id
 
 ### License
 This code is open source software licensed under the [Apache 2.0 License]("http://www.apache.org/licenses/LICENSE-2.0.html").
+
```

Comment:
Can you add a line for ## All tests and checks above the text you have added here?